lib/stripe/plans.rb

```diff
@@ -12,15 +12,19 @@ module Plans
                     :interval,
                     :interval_count,
                     :metadata,
-                    :name, 
+                    :name,
                     :nickname,
                     :product_id,
                     :statement_descriptor,
+                    :tiers,
                     :tiers_mode,
+                    :transform_usage,
                     :trial_period_days,
                     :usage_type
 
-      validates_presence_of :id, :amount, :currency
+      validates_presence_of :id, :currency
+      validates_presence_of :amount, unless: ->(p) { p.billing_scheme == 'tiered' }
+      validates_absence_of :transform_usage, if: ->(p) { p.billing_scheme == 'tiered' }
 
```

Comment:
let's add another quick one 
```suggestion
       validates_presence_of :tiers_mode, if: ->(p) { p.billing_scheme == 'tiered' }
```